multiqc/utils/search_patterns.yaml

```
@@ -364,7 +365,8 @@ happy:
   fn: "*.summary.csv"
   contents: "Type,Filter,TRUTH"
 htseq:
-  contents: "__too_low_aQual"
+  contents_re: '^\w+\t\d+$'
+  num_lines: 2
```

```suggestion
  contents_re: '^(feature\tcount|\w+\t\d+)$'
  num_lines: 1
```
This pattern is a bit different from the one I suggested. Running this on the files in MultiQC_TestData generated 60 matches (true + false) compared to only 21 matches with the pattern I suggested. Any reason for the change?

---

Oh, no reason - I just played with it a bit and saw it's equivalent - but your version should be faster indeed!